### PR TITLE
Resend statistics announcements missing content id in ES

### PR DESCRIPTION
**Pull request by dwhenry**
This will help ensure all documents in elasticsearch have a content id.

https://trello.com/c/niC1v6h1/90-fix-es-records-without-content-ids

### CODE REVIEW

File: db/data_migration/20170511131236_resend_statistics_announcements_to_rummager.rb [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+/government/statistics/announcements/diagnostic-imaging-dataset-for-september-2015
+/government/statistics/announcements/welsh-health-survey-local-authority-and-health-board-results-2012-and-2013
+/government/statistics/announcements/weekly-road-fuel-prices-19-october-2015
+/government/statistics/announcements/social-care-negligence-cases-in-northern-ireland-2013-to-2014
+/government/statistics/announcements/mental-health-and-learning-disabilities-statistics-monthly-report-final-june-2015-and-provisional-july-2015
+/government/statistics/announcements/children-and-young-peoples-well-being-in-the-uk-october-2015
+/government/statistics/announcements/labour-productivity-1998-2014
+)
+
+rummager_index = StatisticsAnnouncement.last.rummager_index
```
Comment (gpeng) [inline db/data_migration/20170511131236_resend_statistics_announcements_to_rummager.rb]: This is unused

File: db/data_migration/20170511131236_resend_statistics_announcements_to_rummager.rb [hunk 2]
```diff
@@ -0,0 +1,154 @@
+# We want to resend these documents to rummager so that the content_id is added to stored record
+
+statistics_announcement_slugs = %w(
+/government/statistics/announcements/care-information-choices-england-december-2015
```
Comment (gpeng) [inline db/data_migration/20170511131236_resend_statistics_announcements_to_rummager.rb]: Can these be indented?

Comment (dwhenry) [inline db/data_migration/20170511131236_resend_statistics_announcements_to_rummager.rb]: done

File: db/data_migration/20170511131236_resend_statistics_announcements_to_rummager.rb (hunk 1 above)
Comment (dwhenry) [inline db/data_migration/20170511131236_resend_statistics_announcements_to_rummager.rb]: removed

**Pull request merged**